libraries/common/io/console.effekt

```diff
@@ -24,6 +24,11 @@ namespace js {
 
   extern jsNode """
     const readline = require('node:readline');
+    if (process.platform === 'win32') {
+      // Set UTF-8 codepage
+      require('child_process').execSync('chcp 65001');
```

Comment:
Just to make sure, is this call truly necessary? (Can you check: does it work without it?)

Is there an alternate way to call this directly via some call that doesn't `exec` a whole new process?

Reply:
If I do it like this:
```
    if (process.platform === 'win32') {
      // Set UTF-8 codepage
      // require('child_process').execSync('chcp 65001');
      process.stdout.setDefaultEncoding('utf8');
    }
```
It sadly does not work for me, maybe I can find another way of doing it tho :)